src/Serilog.Sinks.PeriodicBatching/Sinks/PeriodicBatching/PeriodicBatchingSink.cs

```diff
@@ -23,31 +24,26 @@ namespace Serilog.Sinks.PeriodicBatching;
 /// <summary>
 /// Buffers log events into batches for background flushing.
 /// </summary>
-public class PeriodicBatchingSink : ILogEventSink, IDisposable, IBatchedLogEventSink
+public sealed class PeriodicBatchingSink : ILogEventSink, IDisposable
 #if FEATURE_ASYNCDISPOSABLE
         , IAsyncDisposable
 #endif
 {
-    /// <summary>
-    /// Constant used with legacy constructor to indicate that the internal queue shouldn't be limited.
-    /// </summary>
-    [Obsolete("Implement `IBatchedLogEventSink` and use the `PeriodicBatchingSinkOptions` constructor.")]
-    public const int NoQueueLimit = -1;
+    // Buffers events from the write- to the read side.
+    readonly Channel<LogEvent> _queue;
 
+    // Used by the write side to signal shutdown.
```

Comment:
maybe a note as to why marking the channel closed can't be used to convey that?
(Only scanning, could be way off)